config/initializers/redis_config.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RedisConfig
```

Comment:
It would be nice to use this also in initializers/sidekiq.rb, now that we have it.

Reply:
Good catch, done.